README.md

```diff
@@ -1 +1,157 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Technology Innovation Summit 2023] <a name="about-project"></a>
+
+**[Technology Innovation Summit 2023]** is website for a technology conference with the big figures of the industry.
+
+Video description: [@loomhandle](https://www.loom.com/share/e22a99199aa54571950387dfaacfa636)
+
+## 🛠 Built With <a name="built-with"></a>
+
+- <a name="tech-stack">HTML</a>
+
+- <a name="tech-stack">CSS</a>
+
+- <a name="tech-stack">JavaScript</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Innovation Technology Summit 2023]** Show informations about a technology conference that will take place soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Technology Innovation Summit](https://tshobohwa.github.io/Technology-Innovation-Summit-2023/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor
+- Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone "https://github.com/Tshobohwa/Technology-Innovation-Summit-2023"
+- cd Technology-innovation-summit
+
+### Install
+
+Install this project with:
+
+- cd my-project
+- gem install
+
+### Usage
+
+To run the project, execute the following command:
+
+- live server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Tshobohwa)
+- Twitter: [@twitterhandle](https://twitter.com/SalomonTshoboh1)
+- LinkedIn: [@linkedIn](https://www.linkedin.com/in/tshobohwa-salomon-4bb457245/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Tshobohwa/Technology-Innovation-Summit-2023/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a start and follow us on our platforms for more upcoming projects.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+I would like also to thank [Creative Commons](http://creativecommons.org/licenses/by-nc/4.0/) for giving me the design of this page.
```

Comment:
- [ ] It would be necessary to mention the name of the author in the acknowledements section:

### EXAMPLE

```
Original design by Cindy Shin on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```